types/react-text-mask/tslint.json

```diff
@@ -0,0 +1,12 @@
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
```

Comment:
Please enable all of these, let me know if you're having trouble with a particular one.